javascript/operators/super.json

```
@@ -25,7 +25,7 @@
               "version_added": false
             },
             "nodejs": {
-              "version_added": true
+              "version_added": "5.0.0"
```

Did this come later? https://node.green/#ES2015-functions-super

```suggestion
              "version_added": "6.0.0"
```